wdl/GenotypeDepthPart2.wdl

```diff
@@ -254,8 +258,9 @@ task GetRegenotype{
     # VF<0.01 filter for missing sample variants
     while read chr start end variant type; do
      varid=$variant: #varid should be single variants
-     num=$(fgrep -m1 $varid ~{cohort_combined_bed}|cut -f8)
-     if ((num<100));then
+     num=$(fgrep -m1 $varid ~{regeno_sample_counts_lookup}|cut -f8)
+     thresh=$((num*100))
+     if ( ((thresh < n_samp)) || ((num <= 3)) );then
```

Comment:
Both 100 and 3 should be parameters, with a default values set in `Module04.wdl` like this: `Float regeno_max_allele_freq = 0.01` and `Int regeno_allele_count_threshold = 3`

Reply:
Done